occu-med-portal/src/index.css

```
@@ -107,4 +135,145 @@
   animation: twinkle var(--duration, 3s) infinite ease-in-out;
   animation-delay: var(--delay, 0s);
+  will-change: opacity, transform;
+}
+
+.star-bright {
+  animation-name: sparkle !important;
+}
+
+/* ── Ambient planet bloom ──────────────────────────────── */
+
+@keyframes ambientBloom {
```

Stylelint (17.7.0)

[error] Expected keyframe name "ambientBloom" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)